guides/source/routing.md

```diff
@@ -1289,19 +1364,18 @@ class Video < ApplicationRecord
 end
 ```
 
-```ruby
-video = Video.find_by(identifier: "Roman-Holiday")
-edit_video_path(video) # => "/videos/Roman-Holiday/edit"
+```irb
+irb> video = Video.find_by(identifier: "Roman-Holiday")
+irb> edit_video_path(video)
+=> "/videos/Roman-Holiday/edit"
 ```
 
-Breaking Up *Very* Large Route File into Multiple Small Ones
--------------------------------------------------------
-
-If you work in a large application with thousands of routes, a single `config/routes.rb` file can become cumbersome and hard to read.
+Breaking Up a Large Route File With `draw`
+-----------------------------------------
```

Comment:
Breaking up large route files is only useful for very large applications (and even then we warn against using it). Inspecting and testing routes seems more common/important. Maybe move this section to the bottom?